### DISCUSSION TITLE
I haven't got my education benefits yet.

### DISCUSSION THREAD

**Discussion by danigarciag22** [community/community#144198] [GitHub Education]
### Hi everyone,

I've finished my education verification, but I got no benefits yet, I want to know the reasons. Thanks.

**Comment by NotIntruder** [comment]
It can take up to 3 days to get your GitHub education benefits. I recently applied for mine and it took up to 3 days after verification was completed.
Hope this helps.

**Comment by davevad93** [comment]
Hi @danigarciag22 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @danigarciag22 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
